src/components/Book.js

```diff
@@ -24,17 +24,40 @@ const Book = (props) => {
         <p className="author">{author}</p>
         <ul className="btn-container">
           <li className="comment-btn">
-            <button type="button">Comment</button>
+            <button type="button">Comment |</button>
           </li>
           <li className="remove-btn">
-            <button type="button" onClick={(e) => deleteHandler(e, id)}>Remove</button>
+            <button type="button" onClick={(e) => deleteHandler(e, id)}>Remove |</button>
           </li>
           <li className="edit-btn">
             <button type="button">Edit |</button>
           </li>
         </ul>
       </div>
-      <div className="progress-btn">Update Progess</div>
+
+      <div className="right-progress">
+        <div className="circular-progress">
+          <svg width="200" height="200">
+            <g transform="rotate(-90 100 100)">
+              <circle r="60" cx="100" cy="100" fill="transparent" stroke="lightgrey" strokeWidth="0.5rem" strokeDasharray="439.8" strokeDashoffset="0" />
+              <circle r="60" cx="100" cy="100" fill="transparent" stroke="#0290ff" strokeWidth="0.5rem" strokeDasharray="339.8" strokeDashoffset="66" />
+            </g>
+          </svg>
+          <div>
+            <p className="percentage">64%</p>
+            <p className="complete">Completed</p>
+          </div>
+        </div>
+
+        <div className="vertical-line" />
+        <div className="right-update">
+          <div className="chapter">CURRENT CHAPTER</div>
+          <div className="chapter-number">CHAPTER 17</div>
+          <div className="progress-btn">Update Progess</div>
```

Comment:
Zephin Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/69638013/174438164-60323403-3c1b-46ae-80b5-23b776a205b4.png)  |  ![Screenshot_258](https://user-images.githubusercontent.com/69638013/174438171-4a5187eb-c13e-4255-a0de-1bf695ca7dd8.png)

- Good job so far 💪🏿 👏🏿 👍🏿 

- [ ] Kindly make the Update progress text in your design to be the same color as the text in the zephin design 👍🏿